string.c

```diff
@@ -3630,6 +3630,55 @@ rb_str_concat_multi(int argc, VALUE *argv, VALUE str)
     return str;
 }
 
+static VALUE
+rb_str_append_bytes(VALUE str, VALUE str2)
```

Comment:
May want to add a comment explaining what this function does.

Reply:
Yeah, it's a public method, I'm planning to add doc.